### DISCUSSION TITLE
[Suggestions] Combat logs, provide more details

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#3482] [Ideas]
In fheroes2, it would be nice to provide more details in the combat logs.

Specify the number of Vampire Lords reanimated after one of their attacks:

"Vampire Lords do 112 damage. 1 Bone Dragon perishes. 2 Vampire Lords reanimated."

Specify the target of the Ballista and Turrets when no troops are decimated:

"Ballista does 30 damage to Vampire Lords." 
"Right Turret does 14 damage to Vampire Lords."

Specify the target of the creature troop when no creature is decimated:

"Royal Mummies do 7 damage to Vampire Lords."

In fheroes2:

![More detailed logs 01](https://user-images.githubusercontent.com/43583994/119234442-3b237000-bb2e-11eb-86c6-1302bcaa503f.png)

**Comment by Branikolog** [comment]
Hi, @LeHerosInconnu .
Nice suggestion!
With the above options log would become much more informative.

**Comment by Alucard648** [comment]
"Ghosts do 346 damage. 1 Green Dragon perishes. 1 soul drained"
"Catapult destroys Drawbrige"
"Sandro casts Earthquake. 5 structures destroyed"
"Archibald casts Mass Curse. 3 stacks affected, 1 stack resisted."
"GENIES DEVASTATE HALF OF ENEMY STACK!! 120 Bone Dragons perished"

**Reply by LeHerosInconnu** [reply]
Hello @Alucard648,

Yes, of course, details should be brought to all actions.
The examples given in the original post are just in relation to the screenshot included. :)
